### ISSUE TITLE
Failure with templated yaml file

### ISSUE THREAD

**Issue by julienvey** [adrienverge/yamllint#3]
yamllint fails when using the following template
```yaml
{{key}}: value
```
```python
Traceback (most recent call last):
  File "/usr/bin/yamllint", line 9, in <module>
    load_entry_point('yamllint==1.2.0', 'console_scripts', 'yamllint')()
  File "/usr/lib/python3/dist-packages/yamllint/cli.py", line 110, in run
    for problem in linter.run(f, conf):
  File "/usr/lib/python3/dist-packages/yamllint/linter.py", line 101, in _run
    for problem in get_costemic_problems(buffer, conf):
  File "/usr/lib/python3/dist-packages/yamllint/linter.py", line 72, in get_costemic_problems
    context[rule.ID]):
  File "/usr/lib/python3/dist-packages/yamllint/rules/indentation.py", line 434, in check
    assert context['stack'][-1].type == KEY
AssertionError
```

I would expect to see maybe an error instead, so yamllint can continue with the remaining files

**Comment by adrienverge** [comment]
Thanks @julienvey!

I guess the `{{key}}` is detected as an inline dict and confuses pyyaml. But anyway, yamllint shouldn't crash like this.

I'll investigate as soon as possible :)

**Comment by adrienverge** [comment]
@julienvey The branch on #4 should solve the problem!

Can you tell me if everything is OK for you now, so I can release a new version?

**Comment by adrienverge** [comment]
Released version 1.2.1.
